packages/oranger/oranger.2.0.2/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+authors: ["Francois Berenger"]
+homepage: "https://github.com/UnixJunkie/oranger"
+bug-reports: "https://github.com/UnixJunkie/oranger/issues"
+dev-repo: "git+https://github.com/UnixJunkie/oranger.git"
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+install: [
+  ["sh" "install_ranger.sh"]
+]
+depends: [
+  "dune" {>= "1.0.0"}
+  "conf-cmake" {build}
+  "conf-wget" {build}
```

Comment:
I don't see any use of wget (and it would be refused by the sandbox anyway if it's used at build time). Is this some sort of runtime dependency or is it just a reminiscent of your local `install_ranger.sh` script (in which case it would be safe to remove)

Reply:
yes, probably no more used